src/pages/Filing/UpdateFinancialProfile/TypesFinancialInstitutionSection.tsx

```diff
@@ -95,8 +95,7 @@ function TypesFinancialInstitutionSection({
         {...register('sbl_institution_types_other', {
           value: typeOtherData?.details,
         })}
-        helperText='You must enter a type of financial institution in the text field when
```

Comment:
Aren't these two the same, with different spacing? 👀

Reply:
Yes